src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -790,6 +799,10 @@
     text: |
         az storage blob incremental-copy start --account-key 00000000 --account-name MyAccount --destination-blob MyDestinationBlob --destination-container MyDestinationContainer --source-account-key MySourceKey --source-account-name MySourceAccount --source-blob MyBlob --source-container MySourceContainer --source-snapshot MySnapshot
     crafted: true
+  - name: Copy an incremental copy of a blob asynchronously. (autogenerated)
+    text: |
+        az storage blob incremental-copy start --connection-string myconnectionstring --destination-blob MyDestinationBlob --destination-container MyDestinationContainer --source-uri https://storage.blob.core.windows.net/photos
```

Comment:
blob name and container name should be lowercase

Reply:
```suggestion
        az storage blob incremental-copy start --connection-string myconnectionstring --destination-blob mydestinationblob --destination-container MyDestinationContainer --source-uri https://storage.blob.core.windows.net/photos
```